spring-petclinic-api-gateway/src/main/resources/application.yml

```diff
@@ -33,3 +33,11 @@ spring:
     activate:
       on-profile: docker
     import: configserver:http://config-server:8888
+
+
+
+---
+management:
```

Comment:
It looks like this common configuration may be moved to the https://github.com/spring-petclinic/spring-petclinic-microservices-config/blob/main/application.yml
Are you agree?

Reply:
@MarianConstantinMarica did you see my comment?